File: mobile_apps/Android/store/dropostore/src/main/java/com/dropo/store/models/responsemodel/IsSuccessResponse.java
package com.dropo.store.models.responsemodel;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Created by elluminati on 11-Jan-18.
 */

public class IsSuccessResponse {

    @SerializedName("stripe_error")
    @Expose
    private String stripeError = "";
    @SerializedName("error_code")
    @Expose
    private int errorCode;
    @SerializedName("success")
    @Expose
    private boolean success;
    @SerializedName("message")
    @Expose
    private int message;
    @SerializedName("status_phrase")
    private String statusPhrase;

    public String getStripeError() {
        return stripeError;
    }

    public void setStripeError(String stripeError) {
        this.stripeError = stripeError;
    }

    public int getErrorCode() {
        return errorCode;
    }

    public void setErrorCode(int errorCode) {
        this.errorCode = errorCode;
    }

    public boolean isSuccess() {
        return success;
    }

    public void setSuccess(boolean success) {
        this.success = success;
    }

    public int getMessage() {
        return message;
    }

    public void setMessage(int message) {
        this.message = message;
    }

    public String getStatusPhrase() {
        return statusPhrase;
    }

    public void setStatusPhrase(String statusPhrase) {
        this.statusPhrase = statusPhrase;
    }
}

File: mobile_apps/IOS/store/Store/Models/isSuccessResponse/IsSuccessResponse.swift
//
//  IsSuccessResponse.swift
// Edelivery Store
//
//  Created by tag on 22/02/17.
//  Copyright © 2017 Elluminati. All rights reserved.
//

import Foundation
public class IsSuccessResponse {
    public var success : Bool?
    public var message : Int?
    public var errorCode : Int?
    public var status_phrase : String = ""
    
    required public init?(dictionary: [String:Any]) {
        success = (dictionary["success"] as? Bool) ?? false
        if  success! {
            message = (dictionary["message"] as? Int) ?? 0
        }else {
           errorCode = (dictionary["error_code"] as? Int) ?? 0
        }
        status_phrase = (dictionary["status_phrase"] as? String) ?? ""
    }
}
